media-libs/lcms/lcms-2.8.recipe

```diff
@@ -42,32 +59,32 @@ BUILD_REQUIRES="
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	cmd:gcc${secondaryArchSuffix}
-	cmd:ld${secondaryArchSuffix}
-	cmd:libtoolize
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:aclocal_1.13
 	cmd:autoconf
 	cmd:automake_1.13
```

Comment:
It seems they updated to a newer automake, it might work with cmd:automake also.